docs/backend-requests/custom-session-token.mdx

```diff
@@ -288,15 +275,14 @@ This guide will show you how to customize a session token to include additional
   1. Create the `CustomJwtSessionClaims` interface and declare it globally.
   1. Add the custom claims to the `CustomJwtSessionClaims` interface.
 
-  The following example demonstrates how to add the `fullName` and `primaryEmail` claims to the `CustomJwtSessionClaims` interface.
+  The following example demonstrates how to add the `metadata` claim to the `CustomJwtSessionClaims` interface.
 
   ```tsx {{ filename: 'types/globals.d.ts' }}
   export {}
 
   declare global {
     interface CustomJwtSessionClaims {
-      fullName?: string
-      primaryEmail?: string
+      metadata: string
     }
   }
   ```
```

Comment:
Why make this change? Feels like it's suggesting a clearly riskier option without a clear benefit